cedar-policy/CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 # Changelog
 
 ## 2.4.1
-- Support `Request`s with `Unknown` fields for partial evaluation.
+### Added
+- New API to construct queries with `Unknown` fields for partial evaluation.
```

Comment:
we're adding a new API in a patch release?

Reply:
It's behind the experimental flag for partial evaluation -- should I clarify in the changelog?

Reply:
Maybe something like "New experimental API..."